Fix docs (Duc assigned) (#1614)
pace-neutrons/Horace: documentation/user_docs/docs/user_guide/Interfacing_with_other_programs.rst

Then click "Add" in the next window (you may have to log into your Mathworks account).

If you prefer, you can download the `Horace-Euphonic-Interface` toolbox directly from
`here <https://github.com/pace-neutrons/horace-euphonic-interface/releases>`__
(download the latest ``mltbx`` file). Then run:

.. code-block:: matlab

    matlab.addons.toolbox.installToolbox('/path/to/mltbx_file')

Because `Euphonic` is a Python program, you need to have Python setup on your system,
and to tell Matlab about this.
Please see `here <https://horace-euphonic-interface.readthedocs.io/en/latest/#set-up-python-in-matlab>`__
for more detailed information.
Also, please note that there are restrictions on the supported Python versions for a given
Matlab version.
Please see `here <https://www.mathworks.com/support/requirements/python-compatibility.html>`__
for a table.

In addition, you should also have the `Euphonic` Python package installed in the Python
environment you have set up in Matlab. Please see details in the
`Euphonic installation instructions <https://euphonic.readthedocs.io/en/stable/installation.html>`__.

> We recommend to use the Conda installation.

Reviewer: ```suggestion
We recommend using the Conda installation.
```

> - The ``debye_waller_grid`` parameter is the size of the (Monhkhorst-Pack) :math:`q`-space grid

Reviewer: ```suggestion
- The ``debye_waller_grid`` parameter is the size of the (Monkhorst-Pack) :math:`q`-space grid
```

In addition to ``mat`` and ``selector``, ``horace_sqw`` also takes some other arguments:

- ``'usefast'`` - This tells ``horace_sqw`` to use a faster, more memory efficient but slightly less accurate code than ``spinwave``. In particular, this code achieves its speed gain and lower memory usage by:

>     * Only calculating the perpendicular component ``Sperp`` rather than the full :math:`S^{\alpha\beta}`

Reviewer: ```suggestion
    * Only calculating the perpendicular component :math:`S_{\text{perp}}` rather than the full :math:`S^{\alpha\beta}`
```
Or would this actually be:
```suggestion
    * Only calculating the perpendicular component :math:`S_{\alpha\alpha}` rather than the full :math:`S^{\alpha\beta}`
```

Author: No S_perp is different from the diagonal elements of the tensor.

>       spin-spin correlation tensor, as neutron scattering is only sensitive to ``Sperp``.

Reviewer: As above, so below.

Author: ```suggestion
      spin-spin correlation tensor, as neutron scattering is only sensitive to :math:`S_{\text{perp}}`.
```
